glue_solar/core/core.py

```diff
@@ -0,0 +1,77 @@
+"""
+A reader for sunpy map data.
+
```

Comment:
What was the reason for the new blank line at the end of each doc string?

Reply:
Sorry, it is the style some orgs I have worked with adopts... I can remove them if you prefer

Reply:
I think @astrofrog does it in `glue`, so I am following suit

Reply:
Works for me.

Reply:
Cool